modules.d/99shell-interpreter/module-setup.sh

```diff
@@ -20,3 +20,12 @@ depends() {
     echo "$shell"
     return 0
 }
+
+
+# called by dracut
+install() {
+    if [ ! -e "${initdir}/bin/sh" ]; then
+        inst_multiple bash
+        (ln -s bash "${initdir}/bin/sh" || :)
```

Comment:
This seems wrong to me. It should link to the target that is evaluated in `depends`.